src/popup/pages/Names/AuctionList.vue

```diff
@@ -149,14 +159,21 @@ export default defineComponent({
 @use '../../../styles/typography';
 @use '../../../styles/mixins';
 
+ion-toolbar {
+  --background: var(--screen-bg-color);
+  --min-height: 0;
+
+  padding: 0 var(--screen-padding-x);
+}
```

Comment:
I suggest to match elements by classes. Also the team agreed that there could be only one root class name in the style block.